src/components/ProductGallery/ProductGallery.jsx

```diff
@@ -13,6 +14,40 @@ const ProductGallery = ({
     setShowPopupProductGallery((prevState) => !prevState);
   };
 
+  const livingArea = {
+    id: 1,
+    name: "Living area",
+    image:
+      "https://a0.muscache.com/im/pictures/hosting/Hosting-U3RheVN1cHBseUxpc3Rpbmc6MTEzNDc1NzYxMjc3MDc0NzgxMg%3D%3D/original/f9c21627-6dbb-49b5-9f3f-b57401e8bb1c.jpeg?im_w=1200",
+  };
+  const kitchen = {
+    id: 2,
+    name: "Kitchen",
+    image:
+      "https://a0.muscache.com/im/pictures/hosting/Hosting-U3RheVN1cHBseUxpc3Rpbmc6MTEzNDc1NzYxMjc3MDc0NzgxMg%3D%3D/original/78ed3027-a197-4043-9b7e-8fc79a5425fc.jpeg?im_w=480",
+  };
+
+  const bathroom = {
+    id: 3,
+    name: "Full bathroom",
+    image:
+      "https://a0.muscache.com/im/pictures/hosting/Hosting-U3RheVN1cHBseUxpc3Rpbmc6MTEzNDc1NzYxMjc3MDc0NzgxMg%3D%3D/original/2ca9c23e-85db-48f8-bd21-0718c286dcdf.jpeg?im_w=480",
+  };
+
+  const exterior = {
+    id: 4,
+    name: "Exterior",
+    image:
+      "https://a0.muscache.com/im/pictures/hosting/Hosting-U3RheVN1cHBseUxpc3Rpbmc6MTEzNDc1NzYxMjc3MDc0NzgxMg%3D%3D/original/253a0690-2a1e-4c34-ae7f-968b869be4b5.jpeg?im_w=480",
+  };
+
+  const Bedroom = {
+    id: 5,
+    name: "Bedroom area",
+    image:
+      "https://a0.muscache.com/im/pictures/hosting/Hosting-U3RheVN1cHBseUxpc3Rpbmc6MTEzNDc1NzYxMjc3MDc0NzgxMg%3D%3D/original/5ad7780d-76b5-428f-9219-432243a83a03.jpeg?im_w=480",
+  };
+
```

Comment:
This could be written as as an array of objects: 
const roomsData = [{}, ....  ]

Reply:
Thank you for this comment. I put all objects inside an array.